nvaccess/nvda
# Windows 10 Calculator: give labels to history and memory list items

### Link to issue number:
Fixes #11858

### Summary of the issue:
In Windows 10 Calculator (10.2009 and later), there is no label for history and memory list items.

### Description of how this pull request fixes the issue:
Children of list items do provide necesary components ror memory and history itemss, therefor3e set list item labels based on what the children says.

### Testing strategy:
Manual testing:

1. Open Calculator on Windows 10.
2. Perform some calculations.
3. Select history tab and observe the list item label.
4. Add some expressions to memory (Control+M).
5. Select memory and observe the label of memory entries.

### Known issues with pull request:
None

### Change log entries:
Bug fixes:

In Windows 10 Calculator, NVDA will announce labels for history and memory list items. (#11858)

### Code Review Checklist:



  - description is up to date
  - change log entries
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
